packages/gatsby-codemods/src/transforms/rename-boundactioncreators.js

```diff
@@ -0,0 +1,15 @@
+const OLD_NAME = `boundActionCreators`
+const NEW_NAME = `actions`
+
+module.exports = (file, api, options) => {
```

Comment:
Few comments here!

1. Nit: but can we rename this file to `rename-bound-action-creators.js`? That follows the dasherized naming convention a bit better, e.g. basically replace all capitals with dashes.
2. Can/should we restrict this to certain file names? Is that information available in the `file` argument?

Reply:
2. Do you mean to restrict to `gatsby-node` only? If so this might prove problematic when someone split his `gatsby-node` into multiple files for organization purposes and import those files in `gatsby-node`. Maybe adding note in documentation that running this codemod is best when targeting exact files and not directory path?

Reply:
True! I like your suggestion! Basically just document that this _could_ theoretically replace something you don't want if you run it on the wrong files.